styles.css

```diff
@@ -0,0 +1,144 @@
+body {
+    font-family: "Roboto", sans-serif;
+    font-weight: 400;
+}
+
+.ticket-container {
+    margin: 50px 0 0 50px;
+    width: 782px;
+    height: 273px;
+    background: #EFC684;
+    border-radius: 10px;
+    display: flex;
+}
+
+.main-section {
+    width: 289px;
+}
+
+.ticket-text {
+    text-transform: uppercase;
+    font-size: 55px;
+    margin: 26px 0 0 20px;
+}
+
+.ticket-details-container {
+    display: flex;
+    justify-content: flex-start;
+}
+
+.ticket-details {
+    width: 82px;
+    height: 44px;
+    border: solid 1px black;
+    font-size: 20px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    margin-top: 36px;
+    margin-bottom: 39px;
+}
+
+.ticket-category {
+    margin-right: 30px;
+    margin-left:27px;
+    text-transform: uppercase;
+}
+
+.ticket-price {
+    margin-left: 10px;
+}
+
+.movie-details-container {
+    display: flex;
+    flex-direction: column;
+    margin-right: 52px;
+    justify-content: space-around;
+}
+
+.movie-details {
+    height: 23px;
+}
+
+.movie-hour {
+    margin-top: 26px;
+    width: 100%;
+    border-bottom: 1px solid black;
+}
+
+.movie-location {
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid black;
+}
+
+.gate {
+    width: 82px;
+}
+
+.seat {
+    width: 92px;
+}
+
+.ticket-number {
+    border-bottom: 1px solid black;
+    width: 100%;
+}
+
+.entrance-coupon {
+    border-left: 1px solid;
+    border-left-style: dashed;
+    border-color: #DB0808;
+}
+
+.popcorn-image-container {
+    display: flex;
+    margin: 0 45px 45px 45px ;
+}
+
+.popcorn-cup {
+    margin-top: 45px;
+    position: absolute;
+}
+
+.small-popcorn {
+    position: absolute;
+    margin-left: 25px;
+    margin-top: 50px;
+    opacity: 0;
+}
+
+.popcorn-image-container:hover .small-popcorn {
+    transition: 1s;
+    margin: 0;
+    padding: 0;
+    opacity: 1;
+}
+
+.popcorn-image-container:hover .left-popcorn {
+    transition: 1s;
+    margin-top: 15px;
+    opacity: 1;
+    padding: 20px 0;
+}
+
+.popcorn-image-container:hover .middle-popcorn {
+    transition: 1s;
+    margin-top: 0;
+    opacity: 1;
+    padding: 30px 20px;
+}
+
+.popcorn-image-container:hover .right-popcorn {
+    transition: 1s;
+    margin-top: 15px;
+    opacity: 1;
+    padding: 20px 15px;
+}
+
+.popcorn-cup:hover img{
```

Comment:
There is a missing space here

Reply:
i have used px above because I was basing on figma to avoid differences, no need to change it to % in this case, using % will need to redesign all done now.

Reply:
empty line added at the end

Reply:
width changed from pixels to %